src/markdown-to-partitions.js

```diff
@@ -171,47 +169,8 @@ function _findOuterBreakPoint(lastBreakPoint, index, markdownText) {
   } else if (currentSubstring.startsWith('>>> ') || currentSubstring.startsWith('>>>\n')) {
     index = markdownText.indexOf('>>>', index + 1);
     return index + 3;
-  } else if (currentSubstring.startsWith('> ')) {
```

Comment:
All this stuff isn't necessary anymore?

Reply:
No, I needed it when p partitions continued until the next partition, but since they end on \n now I don't need to check so many things since they all had the same return.  I still pass all my tests.

Reply:
No, I needed it when p partitions continued until the next partition, but since they end on \n now I don't need to check so many things since they all had the same return. I still pass all my tests.